scripts/upload_file_to_s3.py

```diff
@@ -0,0 +1,39 @@
+import os
+import sys
+from typing import Optional
+
+import boto3
+from botocore.exceptions import ClientError
+
+HYDRA_TEST_BUCKET_NAME = "hydra-test-us-west-2"
+
+
+def upload_to_s3(file_path: str, new_name: Optional[str] = None) -> None:
+    _, file_name = os.path.split(file_path)
+    if not file_name:
+        print("Please pass in a path to a file.")
+        return
+
+    client = boto3.client("s3")
+    try:
+        client.upload_file(
+            file_path, HYDRA_TEST_BUCKET_NAME, new_name if new_name else file_name,
+        )
+    except ClientError as e:
+        print(e)
+        return
+
+    print("Uploaded successfully.")
+
+
+if __name__ == "__main__":
+    print(
+        "Please make sure you've configured your AWS Credential env variable correctly."
+    )
+    args_len = len(sys.argv)
+    if args_len == 1:
+        print("Missing arguments, expecting file path.")
+    elif args_len == 2:
+        upload_to_s3(sys.argv[1])
+    else:
+        upload_to_s3(sys.argv[1], sys.argv[2])
```

Comment:
Looks like the perfect opportunity for a little Hydra app (use `@hydra.main()`)
You can use `$env:AWS_CRED_VARIABLE` for easy access to the environment from your config.